symbology/apply-unique-values-with-alternate-symbols/src/main/java/com/esri/samples/apply_unique_values_with_alternate_symbols/ApplyUniqueValuesWithAlternateSymbolsLauncher.java

```diff
@@ -0,0 +1,9 @@
+package com.esri.samples.apply_unique_values_with_alternate_symbols;
+
+/**
+ * Wrapper required for launching a JavaFX 11 app through Gradle or from a jar.
+ */
+public class ApplyUniqueValuesWithAlternateSymbolsLauncher {
+
+  public static void main(String[] args) { ApplyUniqueValuesWithAlternateSymbolsSample.main(args); }
```

Comment:
this was a nit pick for Oliver too: it doesn't affect how the code runs but just for readability let's separate this out a bit

```suggestion
public class ApplyUniqueValuesWithAlternateSymbolsLauncher {

  public static void main(String[] args) { 
    ApplyUniqueValuesWithAlternateSymbolsSample.main(args); 
  }
}
```